### PR TITLE
Use ASCII decoding explicitly in setup.py

### LINKED ISSUE #291: UnicodeDecodeError - 'ascii' codec can't decode characters in 0.4.4 README.rst
I was attempting to install a library that uses colorama in a Docker container and discovered that this was causing install failures from pip. I was using Python 3.6. I dug deeper and discovered that there are two "·" (dot) characters in the 0.4.4 README.rst that is being read as part of the setup.py. These characters cannot be decoded using 7 bit ASCII. They appear to no longer be present in the master branch, but a new release (or an update to 0.4.4) is likely necessary without these characters.

### PR DESCRIPTION
**Pull request by neirbowj**
Prevents future regression of #291 following 688f6cc

### CODE REVIEW

**Comment by neirbowj** [comment]
Dealt with a style nit.

**Comment by yzgyyang** [comment]
Bump.

**Comment by wiggin15** [comment]
As discussed in #291, README should be ascii and not unicode. Since README is now ascii again, a new release will solve this issue, so we don't really have to change the code.
We may want to set the encoding explicitly to "ascii" to avoid this mistake in the future - I will accept a PR for that.
(by the way, this is discussed in [PEP 597)](https://www.python.org/dev/peps/pep-0597/)

File: setup.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 from __future__ import with_statement
 
+from io import open
```
Comment (wiggin15) [inline setup.py:6]: Out of curiosity - is this for Python 2 compatibility?

Comment (neirbowj) [reply setup.py:6]: Yes.

**Comment by wiggin15** [comment]
Thank you!

**Pull request merged**